### PR TITLE
updates for cake 3.6

### PR DESCRIPTION
**Pull request by cewi**
not sure if I got everything.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	],
 	"require": {
 		"php": ">=5.6",
-		"cakephp/orm": "^3.4",
+		"cakephp/orm": "^3.6",
```
Comment (dereuromark) [inline composer.json]: I will only accept ^3.5 at this time. Please make sure you use anything compatible for this, and check travis matrix.
People will have to turn off deprecations for now.

Comment (dereuromark) [reply composer.json]: Refs https://github.com/cakephp/cakephp/pull/11889

File: tests/TestCase/Model/Behavior/GeocoderBehaviorTest.php [hunk 2]
```diff
@@ -63,11 +64,8 @@ public function tearDown() {
 	 */
 	public function testDistance() {
 		$expr = $this->Addresses->distanceExpr(12, 14);
-		if (env('PREFER_LOWEST')) {
-			$expected = '(6371.04 * ACOS((((COS((((PI() / 2) - RADIANS(((90 - Addresses.lat)))))) * COS(PI()/2 - RADIANS(90 - 12)) * COS(((RADIANS((Addresses.lng)) - RADIANS(:param0))))) + (SIN((((PI() / 2) - RADIANS(((90 - Addresses.lat)))))) * SIN((((PI() / 2) - RADIANS(90 - 12)))))))))';
-		} else {
-			$expected = '(6371.04 * ACOS(((COS(((PI() / 2) - RADIANS((90 - Addresses.lat)))) * COS(PI()/2 - RADIANS(90 - 12)) * COS((RADIANS(Addresses.lng) - RADIANS(:param0)))) + (SIN(((PI() / 2) - RADIANS((90 - Addresses.lat)))) * SIN(((PI() / 2) - RADIANS(90 - 12)))))))';
-		}
+		//$expected = '6371.04 * ACOS(COS(PI()/2 - RADIANS(90 - Addresses.lat)) * COS(PI()/2 - RADIANS(90 - 12)) * COS(RADIANS(Addresses.lng) - RADIANS(14)) + SIN(PI()/2 - RADIANS(90 - Addresses.lat)) * SIN(PI()/2 - RADIANS(90 - 12)))';
```
Comment (dereuromark) [inline tests/TestCase/Model/Behavior/GeocoderBehaviorTest.php]: Whats this?

Comment (cewi) [reply tests/TestCase/Model/Behavior/GeocoderBehaviorTest.php]: I have no Idea why I. Maybe I chnaged it while trying to get things working. Should be reverted. Sorry!

**Comment by dereuromark** [comment]
Any idea why those tests would fail?

> PHP Fatal error:  Interface 'Cake\Validation\ValidatorAwareInterface' not found in ...

**Pull request merged**